src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/EntryPointFactoryInterface.php

```diff
@@ -1,30 +0,0 @@
-<?php
```

Comment:
@fabpot I wasn't very sure about deleting this class after RC2 (class was introduced in 5.1). But as it's experimental, I think this is the correct way? Or should I add a BC layer for this interface.

Reply:
That's fine.